CHANGES.md

```diff
@@ -3,6 +3,12 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.400
+- Update model version v0.0.352
+  - Remove `StatusBoard` `fullname` search parameter.
+- Update model version v0.0.351
```

Comment:
There's no such version (https://github.com/openshift-online/ocm-api-model/releases). Please move the next line under v0.0.352

Reply:
Thanks, @tzvatot.  I'm so sorry for being dense.  

I thought I had to account for the missing version here. https://github.com/openshift-online/ocm-api-model/blob/main/CHANGES.md.